src/dmd/json.d

```diff
@@ -416,13 +416,23 @@ public:
     {
         if (s.isModule())
             return;
-        if (!s.isTemplateDeclaration()) // TemplateDeclaration::kind() acts weird sometimes
+        if (s.isStaticCtorDeclaration())
+        {
+            property("kind", s.isSharedStaticCtorDeclaration()
+                     ? "sharedStaticConstructor" : "staticConstructor");
```

Comment:
The current convention for multiple-word kinds seems to be separating the words by spaces, and not using camel case.

Reply:
Indeed, since we feed "kind" verbatim. Updated.